Code/Mantid/instrument/MARI_Parameters.xml

```diff
@@ -13,6 +13,8 @@
      if None - it is undefined -->
 <parameter name="det_cal_file" type="string">
    <value val="None"/>
+   <description is = "The file which defines proper (calibrated) detector positions.
+                      If property set to None - this file is undefined."/>
```

Comment:
I think the description would be more sensible as simply the value of the description element rather than an attribute, i.e.
```xml
<description>
The file which defines proper (calibrated) detector positions. If property set to None - this file is undefined.
<description>
```